app/controllers/admin/resources_controller.rb

```diff
@@ -23,12 +23,25 @@ def create
     end
 
     def update
-      if @resource.update_attributes(resource_params)
-        redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
-                    notice: 'Resource successfully updated.'
+      if params[:inc].present?
+        @resource.used = params[:inc].to_i == 1 ? (@resource.used - 1) : (@resource.used + 1)
+        flag = @resource.save
       else
-        flash[:error] = "Resource update failed: #{@resource.errors.full_messages.join('. ')}."
-        render :edit
+        flag = @resource.update_attributes(resource_params)
+      end
+      respond_to do |format|
+        if flag
+          format.html do
+            redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
+                        notice: 'Resource successfully updated.'
+          end
+          flash[:notice] = "One #{params[:inc].to_i == 1 ? 'resource freed' : 'more resource used'}"
```

Comment:
For this flash message, you need to think about what  the organizer needs to know, and give the organizer the right information to make his/her life easier. 
I, for example, would care to know **which** resource changed, eg. did I just click on T-shirt size M or size L?

I don't think that this message is very informative or helpful.